src/genie/libs/parser/iosxe/show_interface.py

```diff
@@ -1333,6 +1335,7 @@ def cli(self, interface='', output=None):
             m = p8.match(line)
             if m:
                 ret_dict[intf]['access_vlan'] = m.groupdict()['access_vlan']
+                ret_dict[intf]['access_vlan_name'] = m.groupdict()['access_vlan_name']
```

Comment:
It seems like the regex key name is `dummy`, please update the regex as well to have the key as `access_vlan_name`

https://github.com/CiscoTestAutomation/genieparser/blob/master/src/genie/libs/parser/iosxe/show_interface.py#L1332

Reply:
Strange, thought I had already updated that. Must have been missed in my commit.

Reply:
great!

Reply:
Should be corrected now. Apologies for that.

Reply:
Np, looks good now